content/docs/user-guide/testing/best-practices/_index.md

```diff
@@ -0,0 +1,329 @@
+---
+linkTitle: Best Practices
+title: Test Automation Best Practices
+description: Advice for automating tests in Open 3D Engine (O3DE).
+toc: true
+weight: 600
+---
+
+This page provides advice on how many independent developers can efficiently maintain the **Open 3D Engine (O3DE)** with automated tests. The advice applies both to teams using O3DE who write their own automation, as well as to anyone contributing to O3DE. This document is not intended to serve as an exhaustive source of testing advice. It instead provides baseline definitions, and heuristics for investing in test automation. Each concept only serves as a brief introduction, as numerous in depth articles can be found published by other sources.
+
+## Cheatsheet
+
+### Avoid
```

Comment:
I don't like how this heading renders on the docs site. Debating whether it is a useful enough heading to add a description to, or whether `Avoid` and `Remember` should be promoted to the top level. The lower-level headings may then also need a blurb.